src/main/webapp/app/entities/result/result.component.ts

```diff
@@ -25,6 +36,14 @@ export class ResultComponent implements OnInit, OnChanges {
     readonly PROGRAMMING = ExerciseType.PROGRAMMING;
     readonly MODELING = ExerciseType.MODELING;
 
+    readonly IS_BUILDING = ResultTemplateStatus.IS_BUILDING;
```

Comment:
I personally don't like the approach to assign every enum value to the component.
You could also just assign the enum as a property and use it in the template:

`readonly ResultTemplateStatus = ResultTemplateStatus`

Otherwise the component gets a lot of properties and becomes harder to understand. Also in the template it is not clear what e.g. `LATE` is.

Reply:
Good point, I adjusted my implementation.